spec/employee_data_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'csv'
+require 'rspec'
+require 'byebug'
+
+RSpec.describe "employee data" do
+  def csv_data
+    @array ||= []
+
+    if @array.length.zero?
+      CSV.foreach(("employee_data.csv"), headers: true, col_sep: ",") do |row|
+        @array << row
+      end
+    end
+    @array
+  end
+# Identify employees who joined before 2020 and have a salary above $75,000
+  context "employee information" do
+    def employee_info
+      data = []
+        csv_data.each do |row|
+          if row["JoinDate"] < '2020-01-01' && row["Salary"].to_i > 75000
+            data << {name: row["Name"],
+            join_date: row["JoinDate"],
+            salary: row["Salary"]}
```

Comment:
```suggestion
            data << {
              name: row["Name"],
              join_date: row["JoinDate"],
              salary: row["Salary"]
            }
```

This can be written like ☝️ to be more clear and readable